satpy/tests/writer_tests/test_cf.py

```diff
@@ -273,11 +277,15 @@ def test_bounds(self):
         test_array = np.array([[1, 2], [3, 4]]).reshape(2, 2, 1)
         scn["test-array"] = xr.DataArray(test_array,
                                          dims=["x", "y", "time"],
-                                         coords={"time": [np.datetime64("2018-05-30T10:05:00")]},
+                                         coords={"time": [np.datetime64("2018-05-30T10:05:00", "ns")]},
                                          attrs=dict(start_time=start_time,
                                                     end_time=end_time))
         with TempFile() as filename:
-            scn.save_datasets(filename=filename, writer="cf")
+            with warnings.catch_warnings():
+                # The purpose is to use the default time encoding, silence the warning
```

Comment:
so the default time encoding triggers a warning?

Reply:
Yes, the default of `xarray.coding.times.CFDatetimeCoder` seems to be `days since <epoch>` and the datatype we end up is integers, so fractional days are not possible without the library converting to `float`. I don't know how the time dtype would be set to `float` by us, and this was the only case I saw of this warning so didn't use much time investigating.

Reply:
Ok, so then maybe something for the next iteration of warning smashing.